### PR TITLE
Cluster Linking and Schema Linking examples

### PR DESCRIPTION
**Pull request by tomasalmeida**
Examples of cluster linking and schema linking (used in CSTA decks)

### CODE REVIEW

File: cluster-linking-disaster-recovery/README.md [hunk 1]
```diff
@@ -0,0 +1,512 @@
+# Cluster Linking & Schema linking disaster recovery
```
Comment (davetroiano) [inline cluster-linking-disaster-recovery/README.md:1]: `Schema linking` -> `Schema Linking`

**Review by davetroiano — APPROVED** [review]
A few suggestions here but overall a really nice and clear tutorial!

File: cluster-linking-disaster-recovery/README.md (hunk 1 above)
Comment (tomasalmeida) [reply cluster-linking-disaster-recovery/README.md]: done!

File: cluster-linking-disaster-recovery/README.md [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+```shell
+    docker-compose up -d
+```
+
+Two CP clusters are running:
+
+*  Main Control Center available at [http://localhost:19021](http://localhost:19021/)
+*  Disaster Recovery Control Center available at [http://localhost:29021](http://localhost:29021/)
+*  Main Schema Register available at [http://localhost:8085](http://localhost:8085/)
+*  Disaster Recovery Control Center available at [http://localhost:8086](http://localhost:8086/)
```
[the remark this answers is not in the document]
Comment (tomasalmeida) [reply cluster-linking-disaster-recovery/README.md]: done!

File: cluster-linking-disaster-recovery/README.md [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+
+### Verifying cluster linking is up
+
+```shell
+    docker-compose exec disasterKafka kafka-cluster-links --bootstrap-server disasterKafka:29092 --link main-to-disaster-cl --list
+ ````
+
+Output is similar to `Link name: 'main-to-disaster-cl', link ID: 'CdDrHuV5Q5Sqyq0TCXnLsw', remote cluster ID: 'nBu7YnBiRsmDR_WilKe6Og', local cluster ID: '1wnpnQRORZ-C2tdxEStVtA', remote cluster available: 'true'`
+
+### Verifying consumer group offset is migrated
```
[the remark this answers is not in the document]
Comment (tomasalmeida) [reply cluster-linking-disaster-recovery/README.md:145]: makes sense, done!

File: cluster-linking-disaster-recovery/README.md [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+
+4. Verifying cluster linking is up
+
+```shell
+    docker-compose exec mainKafka kafka-cluster-links --bootstrap-server mainKafka:19092 --link disaster-to-main-cl --list
+ ````
+
+Output is similar to `Link name: 'disaster-to-main-cl', link ID: '-FPTBi8JQnGskNQzbrLLmA', remote cluster ID: 'KxjPLtiZQaWPId1UORsRvg', local cluster ID: '59HpxdWkSLSlnqjXnX_ZIw', remote cluster available: 'true'`
+
+5. Verifying consumer group offset is migrated
```
[the remark this answers is not in the document]
Comment (tomasalmeida) [reply cluster-linking-disaster-recovery/README.md:386]: done!

**Review by Cerchie — APPROVED** [review]

**Pull request merged**